netlify.toml

```diff
@@ -0,0 +1,19 @@
+# This should be used for docs also
+[[headers]]
+  for = "/*"
+  [headers.values]
+    Content-Security-Policy = "default-src https:; script-src https: 'unsafe-inline'; style-src https: 'unsafe-inline'; img-src https: data: blob:"
```

Comment:
- Shouldn't we use `script-src 'self'`? The application shouldn't be loading remote scripts
- Same for styles
- Images may come from auth providers, so we need wider permissions there

Reply:
script-src are inline and we have to calculate nonce of them, which can be problematic for developers. Sometimes we use open source libraries. But defailt-src is `https` which gets applied to all.